FastTimeSeries/src/models/InceptionTime.jl

```diff
@@ -0,0 +1,80 @@
+"""
+    InceptionModule(ni::Int, nf::Int, ks::Int = 40, bottleneck::Bool = true)
+
+TBW
+"""
+function InceptionModule(ni::Int, nf::Int, ks::Int = 40, bottleneck::Bool = true)
+    ks = [ks ÷ (2^i) for i in range(0, stop = 2)]
+    ks = [ks[i] % 2 == 0 ? ks[i] - 1 : ks[i] for i in range(1, stop = 3)]  # ensure odd ks
+    bottleneck = ni > 1 ? bottleneck : false
+
+    bottleneck_block = bottleneck ? Conv1d(ni, nf, 1, bias = false) : identity
+
+    convs_layers =
+        [Conv1d(bottleneck ? nf : ni, nf, ks[i], bias = false) for i in range(1, stop = 3)]
+
+    convs = Chain(bottleneck_block, Parallel(hcat, convs_layers...))
+
+    maxconvpool = Chain(MaxPool((3,), pad = 1, stride = 1), Conv1d(ni, nf, 1, bias = false))
+
+    return Chain(Parallel(hcat, convs, maxconvpool), BatchNorm(nf * 4, relu))
+end
+
+"""
+    InceptionBlock(ni::Int, nf::Int = 32, residual::Bool = true, depth::Int = 6)
+
+TBW
+"""
+function InceptionBlock(ni::Int, nf::Int = 32, residual::Bool = true, depth::Int = 6)
+    inception = []
+    shortcut = []
+
+    for d in range(1, stop = depth)
```

Comment:
Still many wild `range`s floating around ;)

Reply:
Yeah, will fix that in the next commit.